object_detection:
  ros__parameters:
    input_img_topic: color_camera/image_raw
    output_bb_topic: object_detection/img_bb
    output_img_topic: object_detection/img
    model_params:
      detector_type: YOLOv5
      model_dir_path: models/
      weight_file_name: auto_final.onnx
      confidence_threshold : 0.7
      show_fps : 1